docs/hadoop-catalog-index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Hadoop catalog index"
+slug: /hadoop-catalog-index
+date: 2025-01-13
+keyword: Hadoop catalog index S3 GCS ADLS OSS
+license: "This software is licensed under the Apache License version 2."
+---
+
+
```

Comment:
For an index, suggest adding the following content: what's a fileset, what problems it to resolve, the usage scenarios,  the capacities of the fileset.

Reply:
Those contents should be in `[Hadoop catalog overview and features](./hadoop-catalog.md)`